src/components/Footer/ContactDetails/ContactDetails.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { icons } from '../../../config/footer';
+
+import './ContactDetails.scss';
+
+const CN = 'contact-block';
```

Comment:
Is **CN** constant @Nazar-bnm suggestion?

Reply:
Yes

Reply:
> Is **CN** constant @Nazar-bnm suggestion?

do you have any objections to this?

Reply:
No, I was just wondering)